scripts/install/createTables.php

```diff
@@ -21,7 +21,6 @@
 
 use oat\taoOutcomeRds\model\RdsResultStorage;
 use Doctrine\DBAL\Schema\SchemaException;
-use \common_Logger;
```

Comment:
Ok, I understand now thanks to the help from Lionel ! It is a script called from a global scope so indeed, we should remove here.